src/ui/components/chart/types.ts

```diff
@@ -0,0 +1,17 @@
+import type { ChartConfiguration, Plugin } from 'chart.js/auto';
+
+export type ChartPoint<T = unknown> = [number, number, T];
+export type ParsedChartPoint<T = unknown> =
```

Comment:
do we need the fallback to unknown?

Reply:
In some cases we can just work with ChartPoint type in some helpers where we don't need information about an extra field. So I decided that is it possible to use this type without mandatory generic type